src/domain/books/BooksPage.tsx

```diff
@@ -1,7 +1,32 @@
-export const BooksPage = (): JSX.Element => {
+import { booksSelector } from "./booksSlice";
+import { Book } from "./types";
+import { BookCard } from "../../components/BookCard";
+import { ToastContainer} from 'react-toastify'
+import { useAppSelector } from "../../store.hooks";
+
+const BooksPage: React.FunctionComponent = () => {
+  const { books: books, loading } = useAppSelector(booksSelector)
   return (
-    <div>
-      Books Page
+    <div className="home-container">
+      <ToastContainer/>
+      {loading === "loaded" ? (
+        <>
+          <h2>New Books Arrivals</h2>
+          <div className="books">
+            {books &&
+              books?.map((book: Book) => (
+                <BookCard {...book} />
+              )
+              )}
+          </div>
+        </>
+      ) : loading === "loading" ? (
+        <p>Loading...</p>
+      ) : (
+        <p>Unexpected error occured...</p>
+      )}
```

Comment:
1) Toast container should be at the root, ideally inside main.tsx.

2) Avoid nested ternaries. Instead, extract it to a function and use if with early return
`const renderContent = () => {
  if (loading === 'loading') {
   return <></>
  }

 if (loading === 'error') {
  return <></>
 }
 
 return <><</>
}`

3 books is not undefined or null so need to put a guard in front of it.